### ISSUE TITLE
Feature Request: Km/Miles per Vehicle

### ISSUE THREAD

**Issue by kurtcallahan** [akhilrex/hammond#96]
Would love to able to control the mileage type per vehicle. I live overseas and have a car that reads in KMs and in the USA it's miles.

**Comment by AndreasWelch** [comment]
Yes, please!  I have two cars as well, one that is in KM, and another that is in Miles.
